gnomad/resources/grch38/gnomad.py

```diff
@@ -365,6 +366,28 @@ def _public_an_ht_path(data_type: str, version: str) -> str:
     return f"gs://gnomad-public-requester-pays/release/{version}/ht/{data_type}/gnomad.{data_type}.v{version}.allele_number_all_sites.ht"
 
 
+def _public_pext_ht_path(type: str, version: str, gtex_versioin: str) -> str:
```

Comment:
For now let's leave this simple since there is only one version, let's only use that one. We can add versioning if/when there is another pext release.